app/assets/javascripts/angular/components/personProfile/personProfile.scss

```
@@ -7,6 +7,11 @@ person-profile {
 
     display: block;
 
+    .unvalidated {
```

```suggestion
    .invalid {
```
? Or `.error`. Idk this is a trivial comment. `unvalidated` implies you never checked it but then how did you know to make it red?

---

No you're right this makes sense, will change :+1: